recipes/seqan3/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "SeqAn3" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/seqan/{{ name|lower }}/releases/download/{{ version }}/seqan-3.0.0-with-submodules.tar.gz
+  sha256: ba8f0fd5d19289f95823235b69fdc5a12aba3578292463eecf750722e8662bd3
+
+build:
+  number: 0
```

Comment:
Propbably have to add `skip: True  # [osx]` since it uses clang on osx. we currently do not support clang. If, however, it is possible to specify to use gcc on osx, we could build it.

Reply:
We should also add `run_exports` like here: https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml#L13